lexicon.txt

```diff
@@ -237,13 +256,15 @@ sizeof
 slotid
 staticsemaphore
 stdint
+stdlib
 struct
 sublicense
 synchapi
 td
 tls
 todo
```

Comment:
Please remove `todo` from the lexicon.  There should be no `TODO` statements in this code.  More importantly, having `todo` in the lexicon eliminates the ability to block code which introduces them.